iosHyperskillApp/iosHyperskillApp/Sources/Modules/ProjectSelectionList/Views/Grid/Cell/ProjectSelectionListGridCellView.swift

```diff
@@ -0,0 +1,118 @@
+import shared
+import SwiftUI
+
+extension ProjectSelectionListGridCellView {
+    struct Appearance {
+        let spacing = LayoutInsets.defaultInset
+        let textGroupSpacing = LayoutInsets.smallInset
+
+        fileprivate func buttonStyle(isSelected: Bool) -> OutlineButtonStyle {
+            OutlineButtonStyle(
+                borderColor: isSelected ? Color(ColorPalette.overlayBlue) : .border,
+                alignment: .leading,
+                paddingEdgeSet: [],
+                backgroundColor: Color(ColorPalette.surface)
+            )
+        }
+    }
+}
+
+struct ProjectSelectionListGridCellView: View {
+    private(set) var appearance = Appearance()
+
+    let project: ProjectSelectionListFeature.ProjectListItem
+    let isSelected: Bool
```

Comment:
We can't selected project that is already selected, should we use this param with `.disabled()` modifier?